src/ovirtapi/transform.js

```diff
@@ -486,6 +486,7 @@ const Snapshot = {
   toApi ({ snapshot }: { snapshot: SnapshotType }): ApiSnapshotType {
     return {
       description: snapshot.description,
+      persist_memorystate: snapshot.persistMemoryState ? 'true' : 'false',
```

Comment:
Is there any reason for not using `toString()` here? Can `snapshot.persistMemoryState` be other than boolean?